### PR TITLE
add a few comments in SDLoraManager

### PR DESCRIPTION
**Pull request by catwell**

### CODE REVIEW

File: src/refiners/foundationals/latent_diffusion/lora.py [hunk 1]
```diff
@@ -223,6 +223,8 @@ def scales(self) -> dict[str, float]:
 
     @staticmethod
     def pad(input: str, /, padding_length: int = 2) -> str:
+        # make all numbers the same length so they sort correctly,
+        # e.g. foo.3.bar -> foo.03.bar
```
Comment (Laurent2916) [inline src/refiners/foundationals/latent_diffusion/lora.py:227]: This should probably be in the docstring

File: src/refiners/foundationals/latent_diffusion/lora.py [hunk 2]
```diff
@@ -236,8 +238,14 @@ def sort_keys(key: str, /) -> tuple[str, int]:
         # this dict might not be exhaustive
```
Comment (Laurent2916) [inline src/refiners/foundationals/latent_diffusion/lora.py:238]:
```suggestion
        """Compute the score of a key, relatively to its suffix.

        When used by [`sorted`][sorted], the keys will only be sorted "at the suffix level".
        
        Args:
            key: The key to sort.

        Returns:
            The padded suffix of the key.
            The score of the key's suffix.
        """
        # this dict might not be exhaustive
```

File: src/refiners/foundationals/latent_diffusion/lora.py [hunk 3]
```diff
@@ -236,8 +238,14 @@ def sort_keys(key: str, /) -> tuple[str, int]:
...
         patterns = ["_{}", "_{}_lora"]
+
+        # apply patterns to the keys of suffix_scores
         key_char_order = {f.format(k): v for k, v in suffix_scores.items() for f in patterns}
+
+        # get the suffix and score for `key` (default: no suffix, highest score = 5)
         (sfx, score) = next(((k, v) for k, v in key_char_order.items() if key.endswith(k)), ("", 5))
+
+        # return tuple of: (padded key prefix, score)
         return (SDLoraManager.pad(key.removesuffix(sfx)), score)
```
Comment (Laurent2916) [inline src/refiners/foundationals/latent_diffusion/lora.py:249]:
```suggestion
        padded_key_prefix = SDLoraManager.pad(key.removesuffix(sfx))
        return (padded_key_prefix, score)
```

File: src/refiners/foundationals/latent_diffusion/lora.py (hunk 1 above)
Comment (catwell) [reply src/refiners/foundationals/latent_diffusion/lora.py:227]: Actually I was considering making this method private instead. It's just a helper for `sort_keys` I don't want it in the docs. WDYT?

**Review by Laurent2916 — APPROVED** [review]

**Pull request merged**